x-pack/test_utils/jest/config.js

```diff
@@ -28,7 +28,7 @@ export default {
   ],
   coverageDirectory: '<rootDir>/../target/kibana-coverage/jest',
   coverageReporters: ['html'],
-  moduleFileExtensions: ['js', 'json', 'ts', 'tsx'],
+  moduleFileExtensions: ['js', 'json', 'ts', 'tsx', 'node'],
```

Comment:
I updated this to be consistent with `src/dev/jest/config.js`, this change isn't needed at this time.